src/main/resources/utilities/icing.properties

```diff
@@ -39,7 +39,10 @@ regex.in.phone.mobile=
 
 test.prop=TEST PROPERTY
 
+utilities.FileUtility.bgthread.watchservice.switch=true
 utilities.I18NUtility.resourcebundle.default=I18N_Resource_Bundle
 utilities.ShellUtility.linux.which=which %s
 utilities.ShellUtility.windows.where=where %s
-utilities.ShellUtility.timeout=5
\ No newline at end of file
+utilities.ShellUtility.timeout=5
+
+test.FileUtilityTest.thread_sleep=1000
```

Comment:
What is this sleep for? What is the unit? Please rename it to contain that info.